scripts/west_commands/build.py

```diff
@@ -264,6 +249,22 @@ def do_run(self, args, remainder):
             except Exception as e:
                 self.wrn(f'Failed to create info file: {build_info_file},', e)
 
+        board, origin = self._find_board()
+
+        # Parse testcase.yaml or sample.yaml files for additional options.
+        if self.args.test_item:
+            # we get path + testitem
+            item = os.path.basename(self.args.test_item)
+            if self.args.source_dir:
+                test_path = self.args.source_dir
+            else:
+                test_path = os.path.dirname(self.args.test_item)
+            if test_path and os.path.exists(test_path):
+                self.args.source_dir = test_path
+                if not self._parse_test_item(item, board):
+                    self.die("No test metadata found")
+            else:
+                self.die("test item path does not exist")
```

Comment:
Nit: add a newline after this entire block

```suggestion
                self.die("test item path does not exist")

```